src/utils/updateConfigurations.js

```diff
@@ -206,6 +296,26 @@ const updateNumerator = (numerators, updatedNumerator) => {
     return numerators
 }
 
+const updateDenominatorRelation = (
+    denominatorRelations,
+    updatedDenominatorRelation
+) => {
+    const currentDenominatorRelation = denominatorRelations.find(
+        (denominatorRelation) =>
+            denominatorRelation.code === updatedDenominatorRelation.code
+    )
+    if (currentDenominatorRelation) {
```

Comment:
rather than updating the current relation, you can just replace it.
Example that Kai wrote: https://github.com/hisprwanda/who-data-quality-annual-report/blob/ca83396ad63866c43050bf6dda3d46a9da095da4/src/utils/configurations/configurationsReducer.js#L56-L60

In general, it's better to keep your functions pure, i.e. not have them update the parameters that you've passed to the function as that tends to cause bugs.